controller/registries/consul/config_test.go

```diff
@@ -50,28 +55,45 @@ func TestStart(t *testing.T) {
 		logger: log.NewLogger(&log.RegistryLoggerOptions{
 			Name: "test",
 		}),
-		softDeletedServices: map[consulService]bool{},
-		done:                make(chan struct{}),
-		watchingServices:    map[consulService]bool{},
-	}
-	config := &consul.Config{
-		ServerUrl: "http://127.0.0.1:8500",
+		done: make(chan struct{}),
 	}
 
-	err := reg.Start(config)
-	assert.NoError(t, err)
+	Convey("Test Start method", t, func() {
```

Comment:
Is there a strong reason to introduce a new testing library and write tests in another style?